apps/docs/src/packages/figma-utils.md

```diff
@@ -0,0 +1,82 @@
+---
+outline: [2, 3]
+---
+
+<script lang="ts" setup>
+import packageJson from "../../../../packages/figma-utils/package.json";
+</script>
+
+# @sit-onyx/figma-utils
+
+{{ packageJson.description }}.
+
+## Use as CLI <Badge text="recommended" />
+
+For a list of supported commands and options, run:
+
+```sh
+npx @sit-onyx/figma-utils@latest --help
+```
+
+::: tip Usage in CI
+If you are using the CLI in CI (e.g. via GitHub actions or Azure pipelines), we recommend that you specify the major version that you want to
+use to prevent possible future breaking changes that might break your pipeline. Example:
+
+```sh
+npx @sit-onyx/figma-utils@1 --help
+```
+
+:::
+
+## Use as npm package
+
+If you don't want to use the CLI, this package also provides utility functions for importing data from the Figma API.
+This is useful if you want customize the CLI commands to have full control over the output.
+
+Install the npm package with your corresponding package manager:
+
+::: code-group
+
+```sh [pnpm]
+pnpm add @sit-onyx/figma-utils
+```
+
+```sh [npm]
+npm install @sit-onyx/figma-utils
+```
+
+```sh [yarn]
+yarn install @sit-onyx/figma-utils
+```
+
+:::
+
+### Examples
+
+#### Import Figma variables to CSS/SCSS variables
```

Comment:
in this case, your example shows only the css variant, so you might want to adjust the title :)

```suggestion
#### Import Figma variables to CSS variables
```